tests/end2end/conftest.py

```diff
@@ -21,11 +21,15 @@
 from fedbiomed.common.constants import ComponentType, ComponentType
 from fedbiomed.common.utils import CONFIG_DIR
 
-_PORT = 50052
 
-@pytest.fixture(scope='session')
+pytest.server_port = 50052
+
```

Comment:
Unless I am missing something, I don't see why you are creating a new attribute for `pytest`.

I don't think it is good practice to do things that way

Reply:
Hi @ybouilla,

It is to be able to share variable between modules in a test session.

Reply:
Ok I see. But is it something commonly done in Pytest?

For what you want to do, i was thinking about the following alternative:
```
def port():
    global _PORT
    if _PORT not in globals():
        _PORT = 50052
    else:
        _PORT += 1
    return str(_PORT)
```
For me this solution is *slightly* better for it doesnot touch `pytest` module

Reply:
Thank you @ybouilla, I will implement as you suggested. What do you think if I rewrite your suggestion as below? 

```
_PORT = 50051
def port():
    global _PORT
    _PORT += 1
    return str(_PORT)
```

Reply:
Thanks it is even better!